src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj

```diff
@@ -112,8 +112,8 @@
     <PackageReference Include="GeneratedSerializers.Json" Version="1.0.0-dev.30" />
     <PackageReference Include="Uno.Material" Version="1.3.3" />
     <PackageReference Include="Uno.Microsoft.Toolkit.Uwp.DeveloperTools" Version="7.1.11" />
-    <PackageReference Include="Uno.UI" Version="4.2.6" />
```

Comment:
Can we avoid updating Uno? It would be preferable to stay on a stable version.

Reply:
It was just that I saw there was a fix that had been pushed on uno
https://github.com/unoplatform/uno/pull/8367

Alternatively, we could remove the IsSpellCheckEnabled="False" it it fixes the issue.